spec/unit/document_spec.rb

```diff
@@ -195,6 +195,18 @@ def balance_before(account_number, date)
     end
   end
 
+  context "with a series defined" do
+    let(:vouchers) {
+      [
+        build_voucher(series: "X")
```

Comment:
Nitpick (though I see we haven't been consistent): we like to use trailing commas on these per https://github.com/barsoom/devbook/tree/master/styleguide#add-commas-to-the-end-of-multiline-lists-and-hashes